src/Schemas/Scorm2004Edition4Schema.php

```diff
@@ -0,0 +1,265 @@
+                                                                "value" => "previous",
+                                                            ],
+                                                            [
+                                                                "name" => "adlnav:hideLMSUI",
+                                                                "value" => "exit",
+                                                            ],
+                                                            [
+                                                                "name" => "adlnav:hideLMSUI",
+                                                                "value" => "exitAll",
+                                                            ],
+                                                            [
+                                                                "name" => "adlnav:hideLMSUI",
+                                                                "value" => "abandon",
+                                                            ],
+                                                            [
+                                                                "name" => "adlnav:hideLMSUI",
+                                                                "value" => "abandonAll",
+                                                            ],
+                                                            [
+                                                                "name" => "adlnav:hideLMSUI",
+                                                                "value" => "suspendAll",
+                                                            ],
+                                                        ],
+                                                    ],
+                                                ],
+                                            ],
+                                            [
+                                                "name" => "imsss:sequencing",
+                                                "childs" => [
+                                                    [
+                                                        "name" => "imsss:deliveryControls",
+                                                        "attributes" => [
+                                                            "completionSetByContent" => "true",
+                                                            "objectiveSetByContent" => "true",
+                                                        ],
+                                                    ],
+                                                ],
+                                            ],
+                                        ],
+                                    ],
+                                    [
+                                        "name" => "metadata",
```

Comment:
I don't think everyone, who uses this package, should have exactly this metadata. You should generate this part of manifest using external data, not hardcode it.